include/memkind.h

```diff
@@ -407,6 +407,14 @@ void memkind_free(memkind_t kind, void *ptr);
 ///
 void *memkind_defrag_reallocate(memkind_t kind, void *ptr);
 
+///
+/// \brief Verify if path points to DAX-enabled file system
+/// \note STANDARD API
+/// \param path path to specified directory for PMEM kind
+/// \return Memkind operation status, MEMKIND_SUCCESS on success, other values on failure
```

Comment:
this should return a boolean (true/false).

Reply:
I prefer to keep convention with int, see comment in:
http://memkind.github.io/memkind/man_pages/memkind.html
 "All other memkind API’s return 0 upon success and an error code defined in the ERRORS section upon failure"

Reply:
this is IMHO very odd - if you want to keep the ability to return an error, then make this function return a boolean by pointer (`memkind_dir_is_dax(..., &is_dax);`).

Reply:
I agree that the combination of returning results and error handling is odd, but this is still a current mechanism already supported with memkind API.
I prefer to be consistent with this mechanism since it is already present in functions like memkind_check_available() and API return value description. I think we agree that the main intention of memkind_path_is_dax is to get information about the passed path's DAX capabilities. Your alternative solution with an additional parameter is just another way to do it. Still, I would prefer to stay with: 
"int memkind_path_is_dax(const char " "*pmem_dir" )," 
because of the legacy convention of return value in memkind API.

Reply:
ok, then call this `memkind_check_dax_path` or something. Don't assume that people will read a man page, and a natural thing to do when a function is called `x_is_y` is to simply assume it returns a bool.

Reply:
Done with rename to memkind_check_dax_path